pkg/testing/tools/agents.go

```diff
@@ -21,7 +21,7 @@ import (
 
 // GetAgentByHostnameFromList get an agent by the local_metadata.host.name property, reading from the agents list
 func GetAgentByHostnameFromList(client *kibana.Client, hostname string) (*kibana.AgentExisting, error) {
-	listAgentsResp, err := client.ListAgents(kibana.ListAgentsRequest{})
+	listAgentsResp, err := client.ListAgents(context.Background(), kibana.ListAgentsRequest{})
```

Comment:
Would it make more sense for the context to be passed in as the first argument to `GetAgentByHostnameFromList`?

Reply:
That would change another dozen on places, wanted to limit the change.